docs/azure_jumpstart_ag/contoso_supermarket/ai/_index.md

```diff
@@ -0,0 +1,42 @@
+# Running AI at the Edge
+
+## Overview
+
+Contoso prides itself on being technology innovators. In the current market, Contoso's leadership recognizes that maintaining their position as leaders in the retail space requires innovation in every aspect of their operations.
+
+To this end, Contoso has developed an AI mechanism that empowers them to enhance the customer experience in their stores. The objective is to accurately detect the number of people waiting at different checkout lanes and enable proactive actions based on this data.
+
+## Store Manager Experience
+
+To achieve this goal, Contoso leverages their existing CCTV cameras by seamlessly integrating them into the Store Manager interface. This integration enables Store Managers to take immediate action, such as opening or closing lanes, based on the real-time data provided by the cameras. All data processing occurs locally using an edge device, ensuring compliance with corporate policies and safeguarding data privacy.
+
+The "Managers Control Center" serves as the central hub of this experience. From this interface, Store Managers can easily monitor the number of people in each lane. The home page consists of two main areas
+
+- Heatmap. Using the heatmap, the Store Manager can open or close checkout lanes to distribute the traffic of the store, using the Toggles above each lane.
+  
+    ![Store Manager Screenshot Heatmap focus](./img/checkout-heatmap.png)
+
+- Reporting. Contoso is interested in improving their store experience by measuring 2 key metrics: Total people in the store, and Shopper Wait Time. This metrics are shown in the Store Manager Control Center as shown below
+
+    ![Store Manager Reporting Pane](./img/reporting.png)
```

Comment:
add one or more screenshots showing the actual live feed and how it can detect the number of users